cmake_modules/SnappyConfig.h

```diff
@@ -0,0 +1,8 @@
+#ifndef SNAPPY_CONFIG_H
```

Comment:
Like this and SnappyCMakeLists.txt, needs a license header, and possibly an addition to our https://github.com/apache/parquet-cpp/blob/master/LICENSE.txt